jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/resources/js/lib/backbone-amd.js

```diff
@@ -0,0 +1,5 @@
+define('slack/backbone', [
+    'jira/backbone-1.3.3'
```

Comment:
Is this module available in all Jira versions starting with 8.15? Slack plugin needs to support all supported Jira versions starting with 8.15.

Reply:
Yes, it is available.